examples/profileElevation.js

```diff
@@ -1,7 +1,5 @@
-goog.provide('app.profile');
+goog.provide('app.profileElevation');
```

Comment:
In English, the order should be reversed to `elevationProfile`.

Reply:
I've named it like this because I think that you search for "profile" in example, not for elevation. But I can change